public/plugin.ts

```diff
@@ -43,6 +46,13 @@ export class SecurityAnalyticsPlugin
     core: CoreSetup<SecurityAnalyticsPluginStartDeps>,
     { dataSourceManagement }: SecurityAnalyticsPluginSetupDeps
   ): SecurityAnalyticsPluginSetup {
+
+    const mountWrapper = async (params: AppMountParameters, redirect: string) => {
+      const { renderApp } = await import("./security_analytics_app");
+      const [coreStart, depsStart] = await core.getStartServices();
+      return renderApp(coreStart, params, redirect, depsStart, dataSourceManagement);
+    };
+
     core.application.register({
```

Comment:
Can we add workspaceAvailability for all the new features of security analytics.

Reply:
addressed